rust/trace-dump/src/chrome_trace.rs

```diff
@@ -216,44 +46,27 @@ impl XiTryFrom<ChromeTraceArrayEntries> for Vec<Sample> {
 /// ```norun
 /// let samples = xi_trace::samples_cloned_sorted();
 /// let mut serialized = Vec::<u8>::new();
-/// chrome_trace::serialize(samples.iter(), OutputFormat::JsonArray, serialized);
+/// chrome_trace::serialize(samples.iter(), serialized);
 /// ```
-pub fn serialize<'a, I, W>(samples: I, _format: OutputFormat, output: W)
+pub fn serialize<'a, W>(samples: &Vec<Sample>, output: W)
```

Comment:
Is there any advantage to calling these functions instead of just using `serde_json` directly?

Reply:
Not much.  I think it's mildly easier to use but probably not worth it.  Should be taken care of when trace-dump is removed.